pallets/committee/src/lib.rs

```diff
@@ -134,10 +136,9 @@ pub trait Trait<I>: frame_system::Trait + IdentityModuleTrait {
 /// Origin for the committee module.
 #[derive(PartialEq, Eq, Clone, Debug, Encode, Decode)]
 pub enum RawOrigin<AccountId, I> {
-    /// It has been condoned by M of N members of this committee.
-    Members(MemberCount, MemberCount),
-    /// Dummy to manage the fact we have instancing.
-    _Phantom(sp_std::marker::PhantomData<(AccountId, I)>),
+    /// It has been condoned by M of N members of this committee
+    /// with `M` and `N` set dynamically in `set_vote_threshold`.
+    Condoned(PhantomData<(AccountId, I)>),
```

Comment:
Condoned has negative annotations attached with it. I'd prefer something more positive like `Approved` or `Endorsed`.